pkg/doc/vc/validate.go

```diff
@@ -39,5 +49,66 @@ func ValidateCredential(
 			errors.New("credential expired"))
 	}
 
+	// Due to the current implementation in AFGO (func verifiable.ParseCredential()),
+	// strict credential validation can only be applied to LDP or unsecured JWT.
+	// Means, that in case first argument in verifiable.ParseCredential() is a JWS -
+	// variable externalJWT will not be empty and validation will be skipped.
+	// So to apply strict validation against JWT/JWS - needs to remove
+	// credential.JWT field, then convert to bytes and explicitly call validation func.
+	if enforceStrictValidation && isJWT(cred) {
+		// If it's SDJWT
+		if credential.SDJWTHashAlg != "" {
+			return validateSDJWTCredential(credential, documentLoader)
```

Comment:
Important:
To apply strict validation against SD JWT, I create display credential first (replace digests in subject with disclosed values) and then apply the validation itself. 
The reason I did so is because claims `_sd`, `_sd_alg` are not passing strict validation (since they are not a part of documents referenced by URI's from @context property).
@sudeshrshetty please confirm it's correct

Reply:
As aligned - the implementation we have right now is fine. No changes needed.